third_party/ngraph/ngraph_tf.BUILD

```diff
@@ -0,0 +1,96 @@
+licenses(["notice"])  # 3-Clause BSD
+
+exports_files(["license.txt"])
+
+filegroup(
+    name = "LICENSE",
+    srcs = [
+        "license.txt",
+    ],
+    visibility = ["//visibility:public"],
+)
+
+load(
+    "@org_tensorflow//tensorflow:tensorflow.bzl",
+    "tf_cc_test"
+)
+
+cc_library(
+    name = "ngraph_libs_linux",
+    srcs = [
+        "lib/libiomp5.so",
+        "lib/libmklml_intel.so",
+    ],
+    visibility = ["//visibility:public"],
+)
+
+cc_library(
+    name = "ngraph_tf",
+    srcs = 
```

Comment:
nit: please put the open bracket on the this line. It'll be useful to run [buildifier](https://github.com/bazelbuild/buildtools/tree/master/buildifier) on all the BUILD files.